File: app/src/main/java/com/lis/player_java/data/room/model/AlbumForMusic.java
package com.lis.player_java.data.room.model;

public class AlbumForMusic {
    public Long albumId;
    public Long albumOwnerId;
    public String accessKey;

    AlbumForMusic(Long albumId, Long albumOwnerId, String accessKey){
        this.albumId = albumId;
        this.albumOwnerId = albumOwnerId;
        this.accessKey = accessKey;
    }
}

File: app/src/main/java/com/lis/player_java/data/room/model/AlbumForMusic.kt
package com.lis.player_java.data.room.model

import androidx.room.PrimaryKey
import com.lis.player_java.data.room.model.AlbumForMusic
import com.lis.player_java.data.room.model.GenreType
import androidx.room.Embedded
import com.lis.player_java.data.room.model.MusicDB
import com.lis.player_java.data.room.model.ArtistDB
import com.lis.player_java.data.room.model.ArtistType

data class AlbumForMusic(val albumId: Long, val albumOwnerId: Long, val accessKey: String)